### PR TITLE
LPS-195619 Convert old AUI in staging-version using JS

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
Hi again, @liferay-staging!

I’v been working in some task removing AUI from the portal to use JS/TS instead and I made some changes for the staging_version.js.
Would you mind to take a look at it an share your thoughts about the changes? Thanks!

## Motivation

Replace old AUI files and try to convert it to JS/TS

[Link to ticket](https://liferay.atlassian.net/browse/LPS-195619)

## Test Scenario 1

1. Activate staging (with page versioning)
2. Create a new Widget Page.
3. Check the behaviour for all kebab menu actions (Page variations, Site variations, History, Redo and Undo).
4. Check the behaviour for the toggle of “Ready for Publish Process”    

https://github.com/liferay-echo/liferay-portal/assets/93203423/5874c316-5494-4bd2-bc4f-9e26cd3a90dd

5. Go to Workflow. Look for “page” and activate Page Revision.
6. Navigate again to the widget page and check the behaviour for the button of “Submit for Workflow”

https://github.com/liferay-echo/liferay-portal/assets/93203423/eca6314f-4ed3-4dec-bdef-438df1b05dff

## Notes

I added a commit with an id for the dropdown to use it in poshi tests (@gaborlovasdotcom) .

### CODE REVIEW

**Comment by gaborlovasdotcom** [comment]
ci:test:staging

**Review by vendeltoreki — APPROVED** [review]
For me, the changes look ok and the manual tests also show no real difference between the old an new versions.
Approved.

File: modules/apps/staging/staging-bar-web/src/main/resources/META-INF/resources/view_layout_revision_details.jsp [hunk 1]
```diff
@@ -149,144 +166,33 @@ else {
...
+						"layoutRevisionStatusURL", layoutRevisionStatusURL
+					).put(
+						"markAsReadyForPublicationURL", markAsReadyForPublicationURL
+					).build()
+				%>'
+				borderless="<%= true %>"
+				displayType="unstyled"
+				dropdownItems="<%= stagingBarDisplayContext.getDropdownItems(layout, layoutRevision, hasWorkflowTask, layoutSetBranch) %>"
+				icon="ellipsis-v"
+				id='<%= liferayPortletResponse.getNamespace() + "stagingVersionOptions" %>'
```
Comment (p2kmgcl) [inline modules/apps/staging/staging-bar-web/src/main/resources/META-INF/resources/view_layout_revision_details.jsp:191]: If we are not using this id in the frontend we can remove it. The component should be identified with some `aria-label` or `label`.

Comment (p2kmgcl) [reply modules/apps/staging/staging-bar-web/src/main/resources/META-INF/resources/view_layout_revision_details.jsp:191]: cc @BarbaraCabrera

**Comment by gaborlovasdotcom** [comment]
cc @david-gutierrez-mesa

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by ealonso** [comment]
Hey @gaborlovasdotcom @vendeltoreki,

Can we forward this one? or Are we waiting for something else?

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by JoyceWang08211** [comment]
ci:test:staging

**Comment by JoyceWang08211** [comment]
> Hey @gaborlovasdotcom @vendeltoreki,
> 
> Can we forward this one? or Are we waiting for something else?

Hi @ealonso, we are waiting for the new test results of ci:test:staging then we will apply xPath updates accordingly. Otherwise, this PR will result in many staging test cases failures.
cc @gaborlovasdotcom

**Comment by JoyceWang08211** [comment]
Continued in https://github.com/JoyceWang08211/liferay-portal/pull/241
I've triggered the forward process.
cc @gaborlovasdotcom and @ealonso

**Pull request closed without merging**